staff/views/job_requests.py

```diff
@@ -23,6 +24,31 @@ class JobRequestList(ListView):
     paginate_by = 25
     template_name = "staff/job_request_list.html"
 
+    def get_context_data(self, **kwargs):
+        orgs = {
+            "is_active": "orgs" in self.request.GET,
+            "items": Org.objects.order_by(Lower("name")),
```

Comment:
Worth eagerly evaluated these queries in the view rather than pushing them in to the template rendering? Given our recent funs...